### PR TITLE
Bug 1150424 - Preventing default on top-panel click and mouse events r=kgrandon

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=672555c722714eb96b1d3ecfcda0b6df537ebeb2

File: apps/system/js/utility_tray.js [hunk 1]
```diff
@@ -49,7 +49,16 @@ window.UtilityTray = {
   blur: function() {},
 
   init: function ut_init() {
-    var touchEvents = ['touchstart', 'touchmove', 'touchend'];
+    var touchEvents = [
+      'touchstart',
+      'touchmove',
+      'touchend',
+      'click',
```
Comment (KevinGrandon) [inline apps/system/js/utility_tray.js]: Why do we need click and mouse* events here? I believe that if we call preventDefault() on the event, we should also prevent the mouse events from firing, correct?

File: apps/system/test/unit/utility_tray_test.js [hunk 2]
```diff
@@ -769,6 +769,54 @@ suite('system/UtilityTray', function() {
...
+      evt.type = 'mousemove';
+      UtilityTray.handleEvent(evt);
+      assert.isTrue(evt.preventDefault.called);
+    });
+
+    test('it should check if preventDefault on mouseup', function() {
+      evt.type = 'mouseup';
+      UtilityTray.handleEvent(evt);
+      assert.isTrue(evt.preventDefault.called);
+    });
```
Comment (etiennesegonzac) [inline apps/system/test/unit/utility_tray_test.js]: Those first 3 tests are a bit confusing... I get the intent but still.

If we just have `mousedown` to handle, I'd like to have just the bottom 3 tests with a `mousedown` event + 1 test with another target where we `assert.isFalse(evt.preventDefault.called)`.

Also if you want the tests to be "more real" you can dispatch real events and check `evt.defaultPrevented` on them.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=60a8158b0b34d5a3fc4a8e4428bb83e9f87ea371

**Pull request merged**